code/GraphQL/LinkablePlugin.php

```diff
@@ -52,43 +58,44 @@ public function apply(ModelQuery $query, Schema $schema, array $config = []): vo
         if ($class !== SiteTree::class && !is_subclass_of($class, SiteTree::class)) {
             return;
         }
-        Schema::invariant(
```

Comment:
With this removed does this still function still `@throws SchemaBuilderException` ?

Reply:
Ah, yup. Good catch.